Source/Core/Core/State.cpp

```diff
@@ -402,7 +402,7 @@ static void CompressAndDumpState(CompressAndDumpState_args& save_args)
   File::IOFile f(temp_filename, "wb");
   if (!f)
   {
-    Core::DisplayMessage("Could not save state", 2000);
+    Core::DisplayMessage("Failed to open temporary state file", 2000);
```

Comment:
```suggestion
    Core::DisplayMessage("Failed to create temporary state file", 2000);
```

Maybe "open" makes sense to a programmer, but to a regular user I think it gives the impression that we're opening a file that already exists.